src/components/Dialogs/GeneralDialog/GeneralDialog.style.ts

```diff
@@ -3,13 +3,17 @@ import { css } from '@emotion/react'
 import { Button } from '@/shared/components'
 import { colors, sizes, breakpoints } from '@/shared/theme'
 
-export const StyledContainer = styled.div`
+type ContainerProps = {
+  wide?: boolean
+}
+
+export const StyledContainer = styled.div<ContainerProps>`
   position: relative;
   width: 90%;
-  max-width: ${sizes(110)};
+  max-width: ${({ wide }) => (wide ? '700px' : '440px')};
```

Comment:
Since I guess `wide` will be a case specific to the multistepper, I think we can just style the dialog directly inside the multistepper instead of adding a prop here. This is an issue at the moment though, `className` isn't passed to the `StyledContainer` so you cannot really style it at the moment. I had to fix that as well when working on the image crop dialog.